src/testfsp3/handler.js

```diff
@@ -381,16 +394,22 @@ exports.postTransfers = async function (req, h) {
           'FSPIOP-Signature': JSON.stringify(fspiopSignature),
           'FSPIOP-HTTP-Method': 'PUT',
           'FSPIOP-URI': fspiopUriHeader,
-          traceparent: req.headers.traceparent ? req.headers.traceparent : undefined,
-          tracestate: req.headers.tracestate ? req.headers.tracestate : undefined
+          traceparent: req.headers.traceparent ? req.headers.traceparent : '',
+          tracestate: req.headers.tracestate ? req.headers.tracestate : ''
         },
-        rejectUnauthorized: false,
-        body: JSON.stringify(transfersResponse)
+        transformRequest: [(data, headers) => {
+          delete headers.common.Accept
+          return data
+        }],
+        httpsAgent: new https.Agent({
+          rejectUnauthorized: false
+        }),
+        data: JSON.stringify(transfersResponse)
       }
       Logger.info(`Executing PUT: [${url}], HEADERS: [${JSON.stringify(opts.headers)}], BODY: [${JSON.stringify(transfersResponse)}]`)
-      const res = await fetch(url, opts)
+      const res = await request(url, opts)
```

Comment:
I dont believe that this would work. axios requires the url to be sent in the opts. therefore your call should look like 
```
const opts = {
    url: '',
    method: 'PUT',
    headers: [],
    ....
}
const res = await request(opts)
```